scheduler/src/main/java/org/cloudfoundry/autoscaler/scheduler/util/DataValidationHelper.java

```diff
@@ -0,0 +1,133 @@
+package org.cloudfoundry.autoscaler.scheduler.util;
+
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.Collections;
+import java.util.List;
+import java.util.TimeZone;
+
+import org.cloudfoundry.autoscaler.scheduler.entity.ScheduleEntity;
+import org.cloudfoundry.autoscaler.scheduler.rest.model.ApplicationScalingSchedules;
+
+/**
+ * 
+ *
+ */
+public class DataValidationHelper {
+
+	/**
+	 * @param object
+	 * @return
+	 */
+	public static boolean isNotNull(Object object) {
+		if (object == null || (object instanceof String && ((String) object).trim().length() == 0))
+			return false;
+		return true;
+	}
+
+	/**
+	 * Checks if the timezone is valid
+	 * 
+	 * @param timeZoneId
+	 * @return
+	 */
+	public static boolean isValidTimeZone(String timeZoneId) {
+		if (isNotNull(timeZoneId)) {
+			String[] validIDs = DateHelper.supportedTimezones;
+			for (String str : validIDs) {
```

Comment:
Why not just define a list version as a private static above with Arrays.asList(DataHelper.supportedTimezones);
Then all you do is a contains check.